layers/+distributions/spacemacs-bootstrap/local/evil-evilified-state/evil-evilified-state.el

```diff
@@ -198,7 +198,11 @@ Needed to bypass keymaps set as text properties."
 (define-key evil-evilified-state-map (kbd "C-d") 'evil-scroll-down)
 (define-key evil-evilified-state-map (kbd "C-u") 'evil-scroll-up)
 (define-key evil-evilified-state-map (kbd "C-o") 'evil-jump-backward)
-(define-key evil-evilified-state-map (kbd "C-i") 'evil-jump-forward)
+(if evil-want-C-i-jump
```

Comment:
use when instead of if

Reply:
> use when instead of if

Indeed, that's better.

> also you can add a test for terminal. so in GUI Emacs, it always binds C-i to evil-jump-forward.

Just to be sure I understand correctly: when in GUI Emacs, you want the value of `evil-want-C-i-jump` to be ignored. I understand the rationale but unless I'm mistaken, this will change behavior from upstream evil.

Finally, is it OK if the pull request contains more commits or should I squash all commits to one?

Reply:
By the way, does `dotspacemacs-distinguish-gui-tab` also play a role here?

Reply:
> Just to be sure I understand correctly: when in GUI Emacs, you want the value of evil-want-C-i-jump to be ignored. I understand the rationale but unless I'm mistaken, this will change behavior from upstream evil.

you're right on this.

I might remove C-i and C-o from evilified mode later and add them on a per major mode basis.

---

no need to squash it manually.
GitHub's default is squash and merge.

Reply:
> By the way, does `dotspacemacs-distinguish-gui-tab` also play a role here?

it doesn't seem to.

Reply:
Done.